sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -834,7 +834,7 @@ func (m *marshaller) expandReshuffle(edge NamedEdge) (string, error) {
 				MergeStatus:     pipepb.MergeStatus_NON_MERGING,
 				WindowCoderId:   windowCoderId,
 				ClosingBehavior: pipepb.ClosingBehavior_EMIT_IF_NONEMPTY,
-				AllowedLateness: 0,
+				AllowedLateness: int64(in.From.WindowingStrategy().AllowedLateness),
```

Comment:
This line needs to be reverted, but it may not need to be?
I'd prefer to keep it explicit since we are basically serializing the old windowing strategy + event times, and writing to a shuffle with a new on in the Global Window. Nothing can be late in the Global Window, but we shouldn't be diverging our definition from Python and Java.

Reply:
I also wasn't sure about it but Java implementation seems to be preserving the allowed lateness value